docs/demography.md

```diff
@@ -301,7 +301,7 @@ demography
 
 :::{warning}
 We use the {meth}`.Demography.isolated_model` function here as a
-convenient means of creating example demographies. However, it's
+convenient mean of creating example demographies. However, it's
```

Comment:
I think "means of" was technically correct. How about we change to "way to", as being a bit clearer?

Reply:
got it!